tests/bench/collection/simple_create_many_test.go

```diff
@@ -19,15 +19,15 @@ import (
 
 func Benchmark_Collection_UserSimple_CreateMany_Sync_0_10(b *testing.B) {
 	ctx := context.Background()
-	err := runCollectionBenchCreateMany(b, ctx, fixtures.ForSchema(ctx, "user_simple"), 0, 10, true)
+	err := runCollectionBenchCreateMany(b, ctx, fixtures.ForSchema(ctx, "user_simple", nil), 0, 10, true)
```

Comment:
thought: It's an optional parameter so it doesn't need to explicitely be set to nil.

Reply:
Just noting that I consider this a part of https://github.com/sourcenetwork/defradb/pull/1925#discussion_r1352853804 and these lines should now be reverted